pymagicc/core.py

```diff
@@ -1069,166 +1142,268 @@ def diagnose_tcr_ecs_tcre(self, **kwargs):
             ],
         )
         # drop all the irrelevant inverse emissions
-        timeseries = timeseries.filter(variable="Inverse Emissions*", level=1, keep=False)
-        tcr, ecs, tcre = self.get_tcr_ecs_tcre_from_diagnosis_results(timeseries)
-        return {"tcr": tcr, "ecs": ecs, "tcre": tcre, "timeseries": timeseries}
+        timeseries = timeseries.filter(
+            variable="Inverse Emissions*", level=1, keep=False
+        )
+        # drop the final year as concs stay constant from some reason,
+        # MAGICC bug...
```

Comment:
Does this have a issue in the MAGICC repo?

Reply:
No, probably should (was trying to avoid an extra distraction..)